SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchNavbar.tsx

```diff
@@ -191,15 +192,17 @@ const EventSearchNavbar = (props: IProps) => {
                 <div className="collapse navbar-collapse" id="navbarSupportedContent" style={{ width: '100%' }}>
                     <div className="navbar-nav mr-auto">
                         <span className="navbar-text">
-                            {timeFilter.date} {timeFilter.time} +/- {timeFilter.windowSize} {formatWindowUnit(timeFilter.timeWindowUnits)}
+                            {newTimeFilter.date} {newTimeFilter.time} +/- {newTimeFilter.windowSize} {formatWindowUnit(newTimeFilter.timeWindowUnits)}
```

Comment:
Does this not need to be checked for `null`?

Reply:
Good catch, moved the check to before this line.